lib/tasks/service_sign_in.rake

```diff
@@ -46,4 +43,39 @@ namespace :service_sign_in do
       puts file
     end
   end
+
+  desc "Unpublish service_sign_in content with type 'gone'"
+  task :unpublish_without_redirect, %i(content_id locale) => :environment do |_, args|
+    USAGE_MESSAGE =
+      "> usage: rake service_sign_in:unpublish_without_redirect[content-id-example,cy]\n".freeze
+
+    content_id = args[:content_id]
+    locale = args[:locale]
+    abort USAGE_MESSAGE unless content_id && locale
+
+    ServiceSignInUnpublishService.call(content_id, locale)
+    puts "> #{content_id} has been unpublished"
+  end
+
+  desc "Unpublish service_sign_in content with type 'redirect'"
+  task :unpublish_with_redirect, %i(content_id locale redirect_path) => :environment do |_, args|
+    USAGE_MESSAGE =
+      "> usage: rake service_sign_in:unpublish_with_redirect[content-id-example,cy,/redirect/path]\n".freeze
```

Comment:
Is it convention to have no spaces between arguments in a rake task?

Reply:
Yes the task tends to break if there are spaces between arguments.

Reply:
Good to know thanks 👍